regression/exceptions/exceptions1/test.desc

```diff
@@ -4,7 +4,7 @@ test.class
 ^EXIT=10$
 ^SIGNAL=0$
 ^.*assertion at file test.java line 26 function.*: FAILURE$
-\*\* 1 of 35 failed (2 iterations)$
+\*\* 1 of 4 failed (2 iterations)$
```

Comment:
Please squash the changes of "Addressed reviewers comments" into the corresponding commits.